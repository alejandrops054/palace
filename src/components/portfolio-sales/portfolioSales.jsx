/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from "react";
import {
  Col,
  Row,
  Input,
  InputGroup,
  InputGroupText,
  Label,
  InputGroupAddon,
  Progress,
  Button,
} from "reactstrap";
import { InfoMembersNoContainer } from "..";
function portfolioSales() {
  return (
    <>
      <Row className="d-flex justify-content-center">
        <Col>
          <h3>Portfolio Sales</h3>
        </Col>
      </Row>
      <Row>
        <Col>
          <Label>New Portfolio: </Label>
          <Input type="select" />
        </Col>
        <Col>
          <Label>File path: </Label>
          <InputGroup>
            <Input type="textarea" />
            <InputGroupAddon addonType="append">
              <InputGroupText>Open File</InputGroupText>
            </InputGroupAddon>
          </InputGroup>
        </Col>
      </Row>
      <Row>
        <Col>
          <Label>Comment:</Label>
          <Input type="textarea" />
        </Col>
      </Row>
      <Row>
        <Col>
          <InfoMembersNoContainer title="Records to process" text="0" />
        </Col>
        <Col className="mt-3">
          <Label>Progress</Label>
          <Progress value="25" />
        </Col>
        <Col>
          <Button>Process</Button>
          <Button>Cancel</Button>
        </Col>
      </Row>
    </>
  );
}

export default portfolioSales;
