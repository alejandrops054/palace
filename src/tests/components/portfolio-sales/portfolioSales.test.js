/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import portfolioSales from '../../../components/portfolio-sales/portfolioSales'

describe('portfolioSales component',() => {
    test('Debe mostrar snapshot', () => {
        
        const wrapper = shallow(<portfolioSales/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})