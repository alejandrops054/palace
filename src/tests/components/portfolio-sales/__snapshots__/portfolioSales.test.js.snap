// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`portfolioSales component Debe mostrar snapshot 1`] = `<portfolioSales />`;
